src/redux/sagas/RatingSaga/RatingSaga.js

```diff
@@ -0,0 +1,34 @@
+import { put, call, takeLatest } from 'redux-saga/effects';
+import axios from 'axios';
+import * as types from '../../actions/RatingsActions/types';
+
+function* ratingWorker({ payload }) {
+  const token = localStorage.getItem('accessToken');
+  const headers = {
+    'Content-Type': 'application/json',
+    Authorization: `TOKEN ${token}`,
+  };
+  try {
+    const apiUrl = 'https://ah-robotics-staging.herokuapp.com/api/v1/rate/this-is-africa/';
+    const response = yield call(axios.post, apiUrl, { user_rating: payload.rating }, { headers });
+    yield put({
+      type: types.RATING_SUCCESS,
+      payload: { rating: response.data },
+    });
+    yield put({
```

Comment:
Why are we dispatching two actions here?

Reply:
The first action adds the rating passed from the components. The second action updates the average rating and feeds the front.